### PR TITLE
feat: deploy via GitHub action

### LINKED ISSUE #1508: Deployment via Travis CI stopped working
My last two commits did not get deployed via Travis CI, see the missing green ✓ on https://github.com/freeCodeCamp/devdocs/commits/master

https://travis-ci.com/github/freeCodeCamp/devdocs/builds says:

> Builds have been temporarily disabled for public repositories due to a negative credit balance. Please go to the Plan page to replenish your credit balance or alter your Consume paid credits for OSS setting.

----

@ojeytonwilliams, we need to find a fix or an alternative!

@jmerle has been experimenting with GitHub Actions here: https://github.com/jmerle/devdocs-ci-testing/tree/master/.github/workflows

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
@simon04 could you give this a quick sanity check?  It's basically what @jmerle suggested, with the `heroku-deploy` action at the end.

If you're happy please merge, then we can see if deployments start happening again.

Fixes #1508.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Build
```
Comment (simon04) [inline .github/workflows/build.yml:1]:
```suggestion
name: Deploy
```

(Also rename the file?)

File: .travis.yml [hunk 2]
```diff
@@ -1,26 +0,0 @@
...
-cache: bundler
-
-before_install:
-  - "echo 'gem: --no-document' > ~/.gemrc"
-  - gem update --system
-  - gem install bundler
-
-script:
-  - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then bundle exec rake; fi
-  - if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then bundle exec thor updates:check --github-token $GH_TOKEN --upload; fi
```
Comment (simon04) [inline .travis.yml:17]: The `thor updates:check` runs once a month and can probably be added later once the deployment is working again.

**Review by simon04 — APPROVED** [review]
Looks good. Let's give it a try! 🚀

File: .travis.yml (hunk 2 above)
Comment (ojeytonwilliams) [reply .travis.yml:17]: Agreed.  I'll add a second action to trigger those on the current schedule.

**Pull request merged**